ci/common/build.sh

```diff
@@ -46,10 +45,10 @@ build_deps() {
 }
 
 prepare_build() {
-  if [[ -n "${CLANG_SANITIZER}" ]]; then
+  if test -n "${CLANG_SANITIZER}" ; then
     CMAKE_FLAGS="${CMAKE_FLAGS} -DCLANG_${CLANG_SANITIZER}=ON"
   fi
-  if [[ "${BUILD_32BIT}" == ON ]]; then
+  if test "x${BUILD_32BIT}" = xON ; then
```

Comment:
Why is the "x" idiom necessary if `"$foo"` is quoted? This works fine in dash:

    test "$foo" = x || echo h